scripts/update-ca-bundle.sh

```diff
@@ -5,6 +5,16 @@
 
 set -e
 
+CERTS_DIR="${1:-/etc/stunnel/certs}"
```

Comment:
This is also being modified by https://github.com/xapi-project/xen-api/pull/4351

It probably would be good to anticipate this and try to avoid any merge conflicts.

Reply:
I've included the updated script also in my xapi PR for cert verification - so we might want to avoid conflicts.